### DISCUSSION TITLE
Mortar _q_point

### DISCUSSION THREAD

**Discussion by srinath-chakravarthy** [idaholab/moose#16171] [Q&A General] [answered]
Hello all, 
I am attempting to retrieve the location of a _qp within the mortar system. @lindayad or anyone else, do you happen to know if this is available, or any other method i can use to get the coordinates. 

Cheers
Srinath

**Comment by aeslaughter** [comment]
You misspelled your @lindsayad

**Comment by lindsayad** [comment, marked as answer]
@srinath-chakravarthy I am assuming that you want the physical location of the quadrature points, not their location on the reference element. The former we cannot do at this moment, but I've opened https://github.com/idaholab/moose/issues/16177 to do so. I'll probably whip that up in the next hour. If you just want the latter, you can do `_qrule_msm->qp(_qp)`

**Comment by srinath-chakravarthy** [comment]
@lindsayad Thanks so much, i was looking for the latter.

**Comment by srinath-chakravarthy** [comment]
Oh sorry was looking for the former :).
